### PR TITLE
fixes issue #278

### PR DESCRIPTION
**Pull request by timball**
senate urls prefer the "www." and many hostnames will not redirect to canonical name w/ the www.

--timball

### CODE REVIEW

**Comment by konklone** [comment]
:+1:

**Comment by JoshData** [comment]
I'm not sure it makes sense to update the historical webpage URLs in legislators-current.yaml. Otherwise +1.

**Comment by konklone** [comment]
Mmmm, good catch. In fact, it's better not to, since their only real value is showing what the URL was at that time. So retroactively updating them is all (though perhaps minor) downside.

@timball, are any of these updates to current terms? If so, can you update the PR to limit it to just current terms?

**Comment by timball** [comment]
the frak are you two talking about ? this is to committees-current.yaml . so ... currently the senate needs www. in the urls . merge this . pedantic whose-a-whats-itz . did not touch any of the "historical" because honestly don't even understand the need for historical thingies to exist . that's why we use git right ? track file changes and evolutions . buttons . magnets . balls . 

--timball

**Comment by timball** [comment]
oh and another things IF the historic files DIDN'T contain the www. then those senate urls would be WRONG . so yeah . they frakking need it . 

--timball

**Comment by JoshData** [comment]
Talkin' 'bout `legislators-current.yaml`.

**Comment by timball** [comment]
OH! this about the "1995" comment . yes . that was irony . used it to reference the prince song "party like it's 1999" . but in web terms , www. feels 1995 to me . regardless all my statements still stand . if the purpose of this repo is to have accurate data then all senate urls need the www. 

--timball

**Comment by timball** [comment]
or maybe my menendez pull .. yeah his thing needs www. too otherwise it's wrong . don't know . don't care . this is what i get for trying to help .

File: legislators-current.yaml [hunk 1]
```diff
@@ -817,7 +817,7 @@
     state: NJ
     party: Democrat
     class: 1
-    url: http://menendez.senate.gov
+    url: http://www.menendez.senate.gov
```
Comment (konklone) [inline legislators-current.yaml]: This edit is good. The other 3 changes above in this file should be reverted. I can submit a PR to the PR.

**Comment by konklone** [comment]
@timball, we've asked lots of people to edit their PRs before merging, it's a normal thing.

The non-current term info for Menendez should keep the (possibly broken) URLs that we logged for them at the time. I'll submit a PR to your PR on your repo, which you can merge and then we'll merge this one.

**Comment by timball** [comment]
it's dumb to hold onto inaccurate and poorly verified bad data . there was never a time that a bare "menendez.senate.gov" worked . and in the weird chance that it did it would have redirected to the www. do whatever you're gonna do but it's dumb . either way merge the other things because accurate is what i'm going for . 

my suspicion is that whomever put that data in there had the hipster www. angst . and just didn't want to see www. dirty their beautifully broken data . that person is bad and should be spanked . this is about accurate data , not beautiful data . EVEN if the the thing redirects you i contend that 'url' should be the url that the user gets too not the one that redirects you away (which btw my PR doesn't actually do the right thing i just noticed that http://www.rules.senate.gov actually redirects you to http://www.rules.senate.gov/public/ so whatever . my whole thing is moot dog shit bite me ) . but even if you go and fix this shit then there will be accurate shit . 

PAX PER IMPERIUM

--timball

**Comment by konklone** [comment]
I filed https://github.com/timball/congress-legislators/pull/1 for you to merge, @timball, which will update this PR in place and clear it for our merge.

**Comment by konklone** [comment]
Thanks for accepting the PR, and for the fixes!

**Comment by JoshData** [comment]
My patience for nonsense ran out. I've locked the thread. Point taken. In the future, when blaming anonymous people, it's worthwhile to run a `git blame` and look up why the change was made in the first place. (In this case, the Menendez URLs date back to my initial commit to this repository.)

**Pull request merged**